packages/client/manifest.json

```diff
@@ -3940,10 +3940,11 @@
         "required": true
       },
       {
-        "type": "multifield",
-        "label": "Allowed filter fields",
+        "type": "allowedFields",
+        "label": "Allowed filter columns",
         "key": "allowedFields",
-        "placeholder": "All fields"
+        "dependsOn": "dataProvider",
+        "nested": true
```

Comment:
`nested` is only used in the context of blocks, and means that settings should not be enriched by the `string-templates` library. We should remove it from here as it serves no purpose.